### PR TITLE
Azure RG enable deploy prerequisites

### PR DESCRIPTION
**Pull request by aydey**

### CODE REVIEW

File: Tasks/DeployAzureResourceGroup/Strings/resources.resjson/en-US/resources.resjson [hunk 1]
```diff
@@ -33,20 +33,72 @@
...
-  "loc.messages.InvalidAction": "This action is not defined. Please check with the author.",
-  "loc.messages.VM_ListFetchFailed": "Task failed while fetching VMs list from '%s' due to the following error: %s",
-  "loc.messages.TaskNotFound": "Task.json file could not be found: %s",
-  "loc.messages.FailedRegisteringEnvironment": "Task failed while registering environment, due to the following error: %s",
-  "loc.messages.selectResourceGroupSuccessfull": "Selected Resource Group: '%s' and stored it in '%s'",
-  "loc.messages.ValidatingCredentialsFailure": "Failed while validating credentials due to following error: %s"
+  "loc.messages.ARG_CreatedResourceGroup": "[Azure Resource Manager]Created resource group '{0}'",
+  "loc.messages.ARG_StartingMachine": "[Azure Resource Manager]Starting machine '{0}'",
+  "loc.messages.ARG_StartedMachine": "[Azure Resource Manager]Started machine '{0}' from Azure provider",
+  "loc.messages.ARG_StoppingMachine": "[Azure Resource Manager]Stopping machine '{0}'",
```
Comment (desattir) [inline Tasks/DeployAzureResourceGroup/Strings/resources.resjson/en-US/resources.resjson]: Delete all PS strings

File: Tasks/DeployAzureResourceGroup/task.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-    "id": "94A74903-F93F-4075-884F-DC11F34058B4",
+    "id": "FB8F4BE5-D18D-42C2-B1D7-D81653960A74",
```
Comment (desattir) [inline Tasks/DeployAzureResourceGroup/task.json]: Why'd you change this?

File: Tasks/DeployAzureResourceGroup/WinRMHttpsListener.ts [hunk 3]
```diff
@@ -0,0 +1,751 @@
+var networkManagementClient = require("azure-arm-network");
```
Comment (desattir) [inline Tasks/DeployAzureResourceGroup/WinRMHttpsListener.ts]: Add entry point to this, in ResourceGroup.ts

File: Tasks/DeployAzureResourceGroup/task.json (hunk 2 above)
Comment (aydey) [inline Tasks/DeployAzureResourceGroup/task.json]: I am reverting this. It was by mistake

**Pull request merged**